agent/consul/state/state_store.go

```diff
@@ -50,6 +50,24 @@ const (
 	watchLimit = 2048
 )
 
+// StoreConfig Handle a StoreConfig to handle particular behaviours
```

Comment:
Not really clear; i guess the start of the comment should have been removed?

Reply:
DONE